.clangd

```diff
@@ -1,32 +1,24 @@
-# https://clangd.llvm.org/config
-
-# Apply a config conditionally to all C files
+# Apply this config conditionally to all C files
 If:
   PathMatch: .*\.(c|h)$
+CompileFlags:
+  Compiler: /home/vyasr/local/rapids/compose/etc/conda/cuda_11.8/envs/rapids/bin/gcc
 
 ---
 
-# Apply a config conditionally to all C++ files
+# Apply this config conditionally to all C++ headers
 If:
-  PathMatch: .*\.(c|h)pp
+  PathMatch: .*\.(c|h)pp$
+CompileFlags:
+  Compiler: /home/vyasr/local/rapids/compose/etc/conda/cuda_11.8/envs/rapids/bin/g++
 
 ---
 
-# Apply a config conditionally to all CUDA files
+# Apply this config conditionally to all CUDA headers
 If:
-  PathMatch: .*\.cuh?
+  PathMatch: .*\.cuh?$
 CompileFlags:
-  Add:
-    - "-x"
-    - "cuda"
-    # No error on unknown CUDA versions
-    - "-Wno-unknown-cuda-version"
-    # Allow variadic CUDA functions
-    - "-Xclang=-fcuda-allow-variadic-functions"
-Diagnostics:
-  Suppress:
-    - "variadic_device_fn"
-    - "attributes_not_allowed"
+  Compiler: /usr/local/cuda/bin/nvcc
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏This might also not be the right location?